model/src/test/kotlin/config/ExcludesTest.kt

```diff
@@ -28,6 +28,7 @@ import com.here.ort.model.PackageReference
 import com.here.ort.model.Project
 import com.here.ort.model.Repository
 import com.here.ort.model.Scope
+import io.kotlintest.TestCase
```

Comment:
Nit: This should be moved to the import block below. Maybe you can fix this retrospectively, @fviernau?